server/controllers/authController/index.js

```diff
@@ -0,0 +1,7 @@
+const AuthController = {
+  show: async (req, res) => {
+    return res.status(200).json({ data: req.app.locals.currentUser })
```

Comment:
this `currentUser` object doesn't have all of the attributes of the user we get from the `UserModel`. What do you think about using the `req.app.locals.currentUser.uid` to query the UserModel for that user so that we keep all the user's consistent?

Reply:
This is untrue. The currentUser has all of the properties. 
https://github.com/AMP-SCZ/dpdash/blob/5d9166607f908fb96fdf762cb55dc41547e7ee00/server/utils/passport/ensure-authenticated.js